### DISCUSSION TITLE
后台样式建议

### DISCUSSION THREAD

**Discussion by appve** [AlistGo/alist#265] [Ideas]
后台写的不好看，layout侧边栏也不会自动收回，有时间抽空写一下前端
前端框架太多了，推荐这个
`[http://layuimini.99php.cn/docs/index.html](http://layuimini.99php.cn/docs/index.html) `

- - -来自 地球网友 的吐槽
你文档编译过程写的马马虎虎，博客也没有写过程，对于没接触golang的人来说自己想改改前端都不会，我自己按照你文档的写的依赖来（旧项目版本），编译出来25-9MB（正常使用），清理残留压缩后也有17MB，而你git编译的6点几兆，当前项目编译出来exe直接3-40多

**Comment by xhofe** [comment]
- 后台只是给自己看的，不满意可以自己写，我只会react
- 编译明确写了只给了解的人准备，不了解直接使用prebuilt
- github上有build.sh编译脚本，明确可以看到是用upx压缩过的
